docs/final.md

```diff
@@ -16,12 +16,12 @@ Data:
-For the server-side programming, we will use the Node.js as part of the backend programming. This is the place for us to store our data in a database. To keep our data up-to-date from the Open Data Prince George’s County website, we will be using the SODA API that is available on the website. Even though the data is coming from PG county’s website, we might use MySQL to redesign our database and this is only when it is necessary. 
-In conclusion, this project requires HTML, CSS, and JavaScript, to develop the structures of our project’s front-end. Then, VS Code, and GitHub as our medium or software to write, edit, collaborate and store our code. For server-side programming, we will be using SODA API, MySQL, and Node.js. However, all of the technology we stated are subject to change due to the complexity of our project.
+We will be using Hypertext Markup Language (HTML) to construct the basic structure of the project and CSS to style. The HTML and CSS will tell the browser how to display the content of our project with different dimensions or scales of the page. However, throughout the process of completing the project, our team will use Javascript for interactiveness. We will also use the Leaflet library for its map functionality, since our goal is to display the locations of various food service facilities in the chosen area on a map. The code will be written in Visual Studio Code (VS Code), a code editor that is redefined and optimized for building and debugging modern web and cloud applications such as GitHub Repository. Additionally, Github will be used to collaborate on the coding aspects. Our targeted devices are personal computers (i.e., laptops, desktop computers) and mobile phones.
+For the server-side programming, we will use the Node.js as part of the backend programming. This is the place for us to store our data in a database. To keep our data up-to-date from the Open Data Prince George’s County website, we will be using the SODA API that is available on the website. Furthermore, we chose to use Heroku CLI to deploy our website. Heroku CLI is used to run applications on a cloud, and allows us to personalize and scale them without affecting functionality. We also chose to use Postman, a tool that is used for API testing. Using it, we were able to test our GET, PUT, and POST requests, and make sure that the data was transferred successfully to the front end.
```

Comment:
Node.js and Express Framework. Express Framework is a Node.js web application server network for building dynamic web applications.

Reply:
What do you mean? Do you want me to include Express Framework to the Technical Systems Design Rationale?